pmd-visualforce/etc/grammar/VfParser.jjt

```diff
@@ -146,7 +146,7 @@ PARSER_END(VfParser)
 	| <GE: ">=" >
 	| <LT: "<" >
 	| <GT: ">" >
-	| <EXCL: ("!"|"~") >
+	| <EXCL: ("!"|"~"|"NOT") >
```

Comment:
The grammar uses the option [IGNORE_CASE](https://javacc.org/javaccgrm#IGNORE_CASE): See [Line 6 of VfParser.jjt](https://github.com/pmd/pmd/blob/master/pmd-visualforce/etc/grammar/VfParser.jjt#L6). So, all tokens are case insensitive.

"~" seems to be an alternative for "!". @sergeygorbaty , is this correct? Can "~" really be used instead of "!" in VisualForce?